compute_endpoint/globus_compute_endpoint/engines/process_pool.py

```diff
@@ -86,7 +86,6 @@ def get_status_report(self) -> EPStatusReport:
                 "min_blocks": 1,
                 "max_workers_per_node": self.executor._max_workers,  # type: ignore
                 "nodes_per_block": 1,
-                "heartbeat_period": self._heartbeat_period,
```

Comment:
Maybe set this to `None` instead so that the report messages are in sync across engines.

Reply:
Done